### ISSUE TITLE
Group and Variable output not matching web link 

### ISSUE THREAD

**Issue by rm17tink** [censusdis/censusdis#287]
**Describe the bug**
If i were to pull he statement below it does not match the website link output:

https://data.census.gov/table/ACSDT5Y2013.B01001?q=B01001:%20Sex%20by%20Age&g=040XX00US36$0500000

**To Reproduce**

```
census_example = ced.download(
         dataset='acs/acs5',
         vintage= 2013,
         # Only one explicitly specified variable.
         download_variables=['NAME'],  
         # Get all the variables that are in this group
         group= 'B01001,
         state=[states.NY],
         county='001'
             )

```

**Expected behavior**
Id expect to have only 1 set of male and female rows for each age group in the year and then the total as shown. However it is producing sub-levels, is my guess? 

**Environment (please complete the following information):**
 - Hardware/Processor: [e.g. windows 10 basic ]

 - Python Version and platform; output of:
```import sys
print(sys.version)
print(sys.platform)
3.11.3 | packaged by Anaconda, Inc. | (main, Apr 19 2023, 23:46:34) [MSC v.1916 64 bit (AMD64)]
win32
```

 - Censusdis Version; output of:
```python
import censusdis
print(censusdis.version)
1.0.1
```

**Additional context**
I am trying to provide a matching structure from the web to the output pulldown for ; someone to accept the vetting process. However it currently is not working.

**Comment by vengroff** [comment]
Yes, this is almost certainly the same issue as #283. I put in a patch for this issue and released https://github.com/censusdis/censusdis/releases/tag/v1.1.13. Default behavior should now match what you saw before the 5/24 release of a new server side from the census bureau. I have confirmed based on some integration tests that started failing on 5/24 that this was indeed when the problem started. If you upgrade to 1.1.13 you should get what you expect. If you do not, please let us know.

The Census Bureau developers are looking into this from their side, but I have not heard back yet. I hope they can revert the behavior because I'm not thrilled with the work around implemented in 1.1.13, but it works in the cases I have tried.

**Comment by vengroff** [comment]
I think I have a confirmed repro of this. Here is the code I used:

```python
import censusdis
import censusdis.data as ced
import censusdis.states as states

df_census_example = ced.download(
     dataset='acs/acs5',
     vintage= 2013,
     # Only one explicitly specified variable.
     download_variables=['NAME'],  
     # Get all the variables that are in this group
     group= 'B01001',
     state=[states.NY],
     county='001'
)

print(f"Number of columns on version {censusdis.version}: {len(df_census_example.columns)}")
```

On version 1.0.1, which you are using I get:

```
Number of columns on version 1.0.1: 332
```

On 1.1.13, after it was patched, I get 

```
Number of columns on version 1.1.13: 53
```

I think this is correct. In this version, 

```python
print(df_census_example.columns)
```

produces

```
Index(['STATE', 'COUNTY', 'NAME', 'B01001_001E', 'B01001_002E', 'B01001_003E',
       'B01001_004E', 'B01001_005E', 'B01001_006E', 'B01001_007E',
       'B01001_008E', 'B01001_009E', 'B01001_010E', 'B01001_011E',
       'B01001_012E', 'B01001_013E', 'B01001_014E', 'B01001_015E',
       'B01001_016E', 'B01001_017E', 'B01001_018E', 'B01001_019E',
       'B01001_020E', 'B01001_021E', 'B01001_022E', 'B01001_023E',
       'B01001_024E', 'B01001_025E', 'B01001_026E', 'B01001_027E',
       'B01001_028E', 'B01001_029E', 'B01001_030E', 'B01001_031E',
       'B01001_032E', 'B01001_033E', 'B01001_034E', 'B01001_035E',
       'B01001_036E', 'B01001_037E', 'B01001_038E', 'B01001_039E',
       'B01001_040E', 'B01001_041E', 'B01001_042E', 'B01001_043E',
       'B01001_044E', 'B01001_045E', 'B01001_046E', 'B01001_047E',
       'B01001_048E', 'B01001_049E', 'GEO_ID'],
      dtype='object')
```

which is just the expected columns without the ones from groups B01001A, B01001B, and so on.

**Comment by vengroff** [comment]
> **Additional context**
> I am trying to provide a matching structure from the web to the output pulldown for ; someone to accept the vetting process. However it currently is not working.

What URL are you trying to match? At https://api.census.gov/data/2013/acs/acs5/groups/B01001.html I see the subgroups coming in, which is not what used to happen before 5/24.

**Comment by rm17tink** [comment]
This has been fixed. I appreciate the support on this as well as all the great work on the project!. 



    Additional context
    I am trying to provide a matching structure from the web to the output pulldown for ; someone to accept the vetting process. However it currently is not working.

What URL are you trying to match? At https://api.census.gov/data/2013/acs/acs5/groups/B01001.html I see the subgroups coming in, which is not what used to happen before 5/24.

I conflated the duplication of rows from the original output and the sourcing. this is a non issue.

 Is it possible to pull in the API website, along with the margin of error for the rows as sourcing from data.() ?
